<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewLearnerDialog</class>
 <widget class="QDialog" name="NewLearnerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Learner</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="loginDataGroupBox">
         <property name="title">
          <string>Login Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="3">
           <widget class="QLabel" name="nicknameCheckLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ITS.qrc">:/Images/Cross.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="nicknameLabel">
            <property name="text">
             <string>&amp;Nickname:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="buddy">
             <cstring>nicknameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>&amp;Password:</string>
            </property>
            <property name="buddy">
             <cstring>passwordLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="passwordCheckLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ITS.qrc">:/Images/Check.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="repeatPasswordLabel">
            <property name="text">
             <string>&amp;Repeat password:</string>
            </property>
            <property name="buddy">
             <cstring>repeatPasswordLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="nicknameLineEdit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="passwordLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="repeatPasswordLineEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="repeatPasswordCheckLabel">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ITS.qrc">:/Images/Check.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="avatarGroupBox">
       <property name="title">
        <string>Avatar</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="avatarLabel">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="ITS.qrc">:/Images/StandardAvatar.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeAvatarPushButton">
          <property name="text">
           <string>Change &amp;Avatar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="genderGroupBox">
     <property name="title">
      <string>Gender</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="indeterminateRadioButton">
        <property name="text">
         <string>&amp;Indeterminate</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="maleRadioButton">
        <property name="text">
         <string>&amp;Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="femaleRadioButton">
        <property name="text">
         <string>&amp;Female</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="descriptionGroupBox">
     <property name="title">
      <string>&amp;Description</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTextEdit" name="descriptionTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ITS.qrc"/>
 </resources>
 <connections/>
</ui>
